dhis-2/dhis-api/src/main/java/org/hisp/dhis/trackedentity/TrackedEntityAttributeService.java

```diff
@@ -170,13 +170,6 @@ List<TrackedEntityAttribute> getTrackedEntityAttributesByDisplayOnVisitSchedule(
    */
   List<TrackedEntityAttribute> getTrackedEntityAttributesDisplayInListNoProgram();
 
-  /**
-   * Get all attributes that user is allowed to read (through program and tracked entity type)
-   *
-   * @return a list of attributes
-   */
-  Set<TrackedEntityAttribute> getAllUserReadableTrackedEntityAttributes();
```

Comment:
Was this one never used?

Reply:
no, it's not used, now we get the attributes from the type of the tracked entity directly.